### DISCUSSION TITLE
Comparatif prise zigbee 3.0 laquelle choisir ? - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#34] [Comments]
# Comparatif prise zigbee 3.0 laquelle choisir ? - Haade.fr

comparatif complet de 4 prises zigbee 3.0, Nous Smart zigbee socket A1Z, Neo Plug-007SPB2, Aqara Smart plug SP-EUC01 et lidl SAPZ 1 A1, dans un monde ou les constructeurs ne manque pas d’innovations dans les prises intelligentes

https://haade.fr/fr/blog/comparatif-prise-zigbee-nous-neo-coolcam-aqara-lidl-laquelle-choisir

**Comment by Bines-185** [comment]
Bonjour  Nico, 
je découvre tes articles qui sont super intéressants, merci pour ton partage.
Sur Jeedom avec plugin et passerelle aqara/xiaomi j'ai décidé de migrer sur gestion Zigbee2MQTT avec une clé Sonof-E firmware 7.3 (et pas 6.10, je l'ai flashée). Mon réseau est pour l'instant peu maillé, instable il faut donc que j'ajoute des répéteurs. Ces prises me paraissent un bon moyen de le faire. J'ai bien trouvé une prise NEO sur AliXXXXXX avec la référence PLUG-001SPB2. et non pas 007. Sais-tu s'il y a une différence ?

**Comment by Breizh13** [comment]
Bonjour,
L'article m'a laissé sur ma faim, car pas très technique. Et la conclusion concernant la prise LIDL m'a laissé surpris, voire pantois car elle dispose de 10 fonctions, rien que ça!!
Je n'ai pas eu l'occasion de la comparer en terme de portée radio, mais elle est d'un excellent rapport qualité prix.

**Comment by nothing-fr** [comment]
Salut à tous,

Personnellement, chez LIDL j'ai trouvé la dernière version de la prise connectée vraiment bien ( https://www.zigbee2mqtt.io/devices/HG08673.html#lidl-hg08673 ), je l'ai eu au même prix que l'ancienne avec la mesure de consommation, du coup je la trouve vraiment inintéressante. Elle marche nickel avec z2m / hass. Ce qui la ramène au niveau de la NOUS niveau fonctionnalités.

Voilà, donc pas forcement à jeter la LIDL... surtout que sur la NOUS, j'ai vu pas mal de retour sur des sifflements de la prise...

Bref, a creuser ! +++

**Reply by nothing-fr** [reply]
typo, pas "initéressante" mais plutôt le contraire...
